wrappers/net/http/handler_wrapper.go

```diff
@@ -0,0 +1,115 @@
+package epsagonhttp
```

Comment:
It would be good to update the Epsagon docs with this + gin support

Reply:
already did